### PR TITLE
Add Hexagonal Architecture library

### PR DESCRIPTION
**Pull request by corlaez**
Adding a library for the Ports and Adapters Architecture, also known as Hexagonal, created by Alistair Cockburn.

### CODE REVIEW

**Comment by dwelle** [comment]
Hey @corlaez some of those items don't seem very suitable as library items. Each item should be generic, reusable, self-contained (though we haven't yet specified it anywhere, so our fault).

For example, the `Configuration and Mock Testing` are just code examples and explanation:

image

Similarly, the `Implementation and Testing`, though you could extract the folders and files and make them into separate lib items for sure.

I'd say it applies to others as well. E.g. here I think the explanation text shouldn't be part of the items:

image

Moreover, some of the items you've included are just taken directly from another library, specifically these two

image

Soon, we will allow to filter and install specific library items, so these would become duplicates.

**Comment by corlaez** [comment]
Thanks for the feedback! I will update accordingly!

**Comment by dwelle** [comment]
Looks good, let's ship it!

File: libraries.json [hunk 1]
```diff
@@ -898,5 +898,19 @@
...
+    "authors": [
+      {
+        "name": "Armando Cordova Pelaez",
+        "url": "https://github.com/corlaez"
+      }
+    ],
+    "source": "corlaez/hexagonal-architecture.excalidrawlib",
+    "preview": "corlaez/hexagonal-architecture.png",
+    "created": "2021-09-21",
+    "updated": "2021-09-21"
```
Comment (dwelle) [inline libraries.json:914]:
```suggestion
    "created": "2021-09-24",
    "updated": "2021-09-24"
```

**Pull request merged**